src/main/java/edu/ksu/canvas/attendance/form/RosterFormValidator.java

```diff
@@ -12,6 +13,8 @@
 @Component
 public class RosterFormValidator implements Validator {
 
+    private static final Logger LOG = Logger.getLogger(RosterFormValidator.class);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "LOG" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1068)